### ISSUE TITLE
[aws-cdk-java] Missing maven dependencies on JCentral

### ISSUE THREAD

**Issue by drissamri** [aws/aws-cdk#9382]
A cleanly generated application doesn't work currently because maven dependencies are not being published correctly anymore. 

### Reproduction Steps
`cdk init app --language=java`

### Error Log
```
[ERROR] Failed to execute goal on project favorites-service-deploy: Could not resolve dependencies for project be.drissamri:favorites-service-deploy:jar:LATEST: Failed to collect dependencies at software.amazon.awscdk:core:jar:1.55.0 -> software.amazon.awscdk:cdk-cloud-assembly-schema:jar:[1.55.0]: No versions available for software.amazon.awscdk:cdk-cloud-assembly-schema:jar:[1.55.0] within specified range -> [Help 1]
[ERROR] 
```
It seems like 1.46.0 is the latest available version on JCenter maven repository: https://jcenter.bintray.com/software/amazon/awscdk/cdk-cloud-assembly-schema/

### Environment

  - **CLI Version      :** 1.55.0
  - **Framework Version:** 1.55.0
  - **Node.js Version:** v12.13.1
  - **OS               :** OSX
  - **Language (Version):** Java (8)

--- 

This is :bug: Bug Report

**Comment by RomainMuller** [comment]
You might have gotten too fast... Maven Central unfortunately becomes consistent pretty slowly for new releases...

**Comment by drissamri** [comment]
The last version available is 1.46.0: https://jcenter.bintray.com/software/amazon/awscdk/cdk-cloud-assembly-schema/
